docs/reviewing.md

```diff
@@ -0,0 +1,37 @@
+# Reviewing policies
+
+The following policies are preferred during the review, but not mandatory:
+
+## Trailing white-spaces
+
+Avoid trailing white-space characters, if possible
+
+## Quotes
+
+If possible, try to avoid mixing single quotes (`'`) and double quotes (`"`) in python code (`conanfile.py`, `test_package/conanfile.py`). Consistency in preferred.
+
+## Order of methods and attributes
+
+Prefer the following order of documented methods in python code (`conanfile.py`, `test_package/conanfile.py`):
+
+- init
+- set_name
+- set_version
+- export
+- export_sources
+- config_options
+- configure
+- requirements
+- package_id
+- build_id
+- build_requirements
+- system_requirements
+- source
+- imports
+- build
+- package
+- package_info
+- deploy
+- test
+
+the order above resembles the execution order of methods on CI. therefore, for instance, `build` is always executed before `package` method, so `build` should appear before the `package` in `conanfile.py`.
```

Comment:
```suggestion
the order above resembles the execution order of methods on CI. therefore, for instance, `build` is always executed before `package` method, so `build` should appear before the `package` in `conanfile.py`.

## Simultaneous conflicting methods

Try to avoid mixing the `requires` field and `requirements()` method in the same `conanfile.py`. It's better to put requirements close to each other.
The same thing goes about `build_requires` and `build_ requirements()`.
```

Reply:
@uilianries created a hook, so this change is not needed.
https://github.com/conan-io/hooks/pull/261